src/pages/Main/TogethderSlider.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useRef } from 'react'
+import { IRandomRecordData } from 'types/recordData'
+import recordIcons from '@assets/record_icons'
+import { useNavigate } from 'react-router-dom'
+import useSwipe from '@hooks/useSwipe'
+
+export default function TogethderSlider({
+  randomRecordData,
+}: {
+  randomRecordData: IRandomRecordData[] | null
+}) {
+  const navigate = useNavigate()
+
+  const dragRef = useRef<HTMLDivElement | null>(
+    null
+  ) as React.MutableRefObject<HTMLDivElement>
+
+  const { handleMouseDown, isDragging, setIsDragging } = useSwipe(dragRef)
+
+  const handleClickRecord = (recordId: number) => {
+    if (isDragging) {
+      setIsDragging(false)
+      return
+    }
+    navigate(`/record/${recordId}`)
+  }
+
+  return (
+    <div
+      className="relative ml-6 flex h-full flex-nowrap overflow-x-hidden"
+      ref={dragRef}
+      onMouseDownCapture={(e) => {
+        handleMouseDown(e)
+      }}
+    >
+      {randomRecordData !== null &&
```

Comment:
아래처럼 적용하면 더욱 족을 것 같아요
```suggestion
      {!randomRecordData &&
```